scripts/diff_stix.py

```diff
@@ -651,6 +711,12 @@ def layers_dict_to_files(outfiles, layers):
         action="store_true",
         help="Add a key explaining the change types to the markdown"
     )
+
+    parser.add_argument("--no-details",
+        type=str,
+        action="append",
+        help="Skip providing details for the specified field"
+    )
```

Comment:
We may also want multiple modes of details. Something like this perhaps:
```
parser.add_argument("--details",
    choices=["fields", "values"]
    help="provide detailed report of what changed for each object. If 'fields' is specified, it will report on what fields changed. If 'values' is specified, it will report on how the value of each field changed as well."
)
```
This will help avoid excessively long outputs for users who just want a bullet-point ("changed: description, data_sources") list for each object instead of the full before/after for each field which might not be nessecary for every use case.